MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,12 +20,17 @@ final class FeedViewModel {
 	
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onErrorStateChange: Observer<String?> = { _ in }
```

Comment:
The `onErrorStateChange` property doesn't need a default value. Just like the other closure properties in this class, it can be optional:

```suggestion
	var onErrorStateChange: Observer<String?>?
```